data/data/vsphere/master/main.tf

```diff
@@ -2,30 +2,37 @@ locals {
   description = "Created By OpenShift Installer"
 }
 
-resource "vsphere_virtual_machine" "vm" {
-  count = var.instance_count
+provider "vsphere" {
+  user                 = var.vsphere_username
+  password             = var.vsphere_password
+  vsphere_server       = var.vsphere_url
+  allow_unverified_ssl = false
+}
+
+resource "vsphere_virtual_machine" "vm_master" {
+  count = var.master_count
 
-  name                 = "${var.cluster_id}-${var.name}-${count.index}"
+  name                 = "${var.cluster_id}-master-${count.index}"
   resource_pool_id     = var.resource_pool
   datastore_id         = var.datastore
-  num_cpus             = var.num_cpus
-  num_cores_per_socket = var.cores_per_socket
-  memory               = var.memory
+  num_cpus             = var.vsphere_control_plane_num_cpus
+  num_cores_per_socket = var.vsphere_control_plane_cores_per_socket
+  memory               = var.vsphere_control_plane_memory_mib
   guest_id             = var.guest_id
   folder               = var.folder
   enable_disk_uuid     = "true"
   annotation           = local.description
 
-  wait_for_guest_net_timeout  = "0"
-  wait_for_guest_net_routable = "false"
+  wait_for_guest_net_timeout = 0
+  wait_for_guest_ip_timeout  = 15
```

Comment:
@e-tienne our dev/ci environments are very high performance. I am concerned with this number. Probably should be at least 30

Reply:
Thanks, I will set it to 30 for now. Do we have other environments to test with?

Reply:
> Do we have other environments to test with?

We have two vSphere CI envs boskos determines where it lands.

Reply:
ok, let's see how it goes. If you think that 30 is not enough, please let me know. 

I'll mark it as resolved for now.